buddyboss/buddyboss-platform
# If you post on a member's timeline, it will disappear.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1525 

### How to test the changes in this Pull Request:

1. Go to a member's profile.
2. Post on Timeline
3. Activity link will appear to go and see because we can't fix it with scope tabs active because it belongs to mentions tab and we have to refresh the page and go to mentions tab.

### Proof Screenshots or Video

https://prnt.sc/un495d

### Other information:
